pilot/pkg/model/destination_rule.go

```diff
@@ -18,61 +18,49 @@ import (
 	"fmt"
 
 	networking "istio.io/api/networking/v1alpha3"
-
-	"istio.io/istio/pkg/config/host"
 )
 
 // This function merges one or more destination rules for a given host string
 // into a single destination rule. Note that it does not perform inheritance style merging.
 // IOW, given three dest rules (*.foo.com, *.foo.com, *.com), calling this function for
 // each config will result in a final dest rule set (*.foo.com, and *.com).
 func (ps *PushContext) combineSingleDestinationRule(
```

Comment:
nit: may be we should call this `mergeDestinationRule`?

Reply:
I am ok